docs/features/software-catalog/catalog-customization.md

```diff
@@ -151,18 +151,18 @@ export const CustomCatalogPage = ({
   return (
     ...
         <EntityListProvider>
-          <FilteredEntityLayout>
-            <FilterContainer>
+          <CatalogFilterLayout>
+            <CatalogFilterLayout.Filters>
               <EntityKindPicker initialFilter="component" hidden />
               <EntityTypePicker />
               <UserListPicker initialFilter={initiallySelectedFilter} />
 +             <EntitySecurityTierPicker />
```

Comment:
spurious addition :)

Reply:
It's just the diff syntax being highlighted 😅

Reply:
ah man